data/json/requirements/toolsets.json

```diff
@@ -120,7 +120,8 @@
     "id": "soldering_standard",
     "type": "requirement",
     "//": "Soldering metal items",
-    "tools": [ [ [ "soldering_iron", 1 ], [ "toolset", 1 ] ] ],
+    "qualities": [ { "id": "SOLDER", "level": 1 } ],
+    "tools": [ [ [ "soldering_iron", 1 ], [ "toolset", 1 ], [ "surface_heat", 2, "LIST" ], [ "lighter", 2 ], [ "ref_lighter", 2 ] ] ],
```

Comment:
Does this actually require you to use either a proper soldering iron or a improvised one by having a fire source? Because I don't see `soldering_iron_improvised` referenced anywhere.